src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -91,16 +91,13 @@ def _get_sections() -> pd.DataFrame:
     return sections
 
 
-def _get_assignments(section_id) -> Callable:
+def _get_assignments(section_id: int) -> Callable:
     # This nested function provides "closure" over `section_id`
     def __get_assignments() -> Optional[pd.DataFrame]:
         assignments = service.get_assignments(section_id)
         result_bucket["assignments"] = assignments
-        return (
-            service.map_assignments_to_udm(assignments, section_id)
-            if not assignments.empty
-            else None
-        )
+
+        return assignments
```

Comment:
I was able to merge the map_to_udm call back into the service facade instead of calling it here in `main`.